.circleci/push_sck_artifacts.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+sudo apt-get install -y python-pip libpython-dev > /dev/null 2>&1
+echo "Installing aws cli..."
+sudo pip install awscli > /dev/null 2>&1
+echo "Pushing SCK artifacts to s3..."
+aws s3 cp helm-artifacts/splunk-kubernetes-logging*.tgz s3://k8s-ci-artifacts/
+aws s3 cp helm-artifacts/splunk-kubernetes-metrics*.tgz s3://k8s-ci-artifacts/
+aws s3 cp helm-artifacts/splunk-kubernetes-objects*.tgz s3://k8s-ci-artifacts/
+aws s3 cp helm-artifacts/splunk-connect-for-kubernetes*.tgz s3://k8s-ci-artifacts/
```

Comment:
can combine all `cp` commands - `aws s3 cp helm-artifacts/splunk-kubernetes-*.tgz s3://k8s-ci-artifacts/`

Reply:
dosen't work aws cli does not interpret it appropriately

Reply:
yeah, might need to use `sync --include<>` instead of `cp`. But it's okay, we don't need to change that.